pkg/util/daemon_prepper.go

```diff
@@ -57,7 +57,9 @@ func (p *DaemonPrepper) GetFileSystem() (string, error) {
 	}
 
 	src, err := ref.NewImageSource(nil)
-	defer src.Close()
```

Comment:
I think you can get the same thing by skipping the if nil check and moving this to under the if err check.